offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectViewModel.groovy

```diff
@@ -106,7 +109,9 @@ class CreateProjectViewModel {
         desiredProjectCode = ""
         resultingProjectCode = ""
         projectCodeValidationResult = ""
-        codeIsValid = false
+        projectCodeIsValid = false
```

Comment:
you need to be carefull with namechanges, also the change by this variable broke the code. But I cannot access it. It would be good if you would use your IDE to change variable names since it delegates the changes in the class to where the variable is referenced.